src/reporter/reporter.py

```diff
@@ -147,15 +147,26 @@ def notify():
         fiware_sp = request.headers.get('fiware-servicepath', None)
     else:
         fiware_sp = None
-
-    # Send valid entities to translator
-    with translator_for(fiware_s) as trans:
-        trans.insert(payload, fiware_s, fiware_sp)
-
-    msg = 'Notification successfully processed'
+    for entity in payload:
+        attrs = list(iter_entity_attrs(entity))
```

Comment:
isn't this getting the attributes of the last entity in the payload? e.g. if

    payload = [ { a1{...}, a2{...} } , { a2{...}, a3{...} } ]

then `attrs = a2, a3` ? If I understand your algo correctly, this should cause a problem down below in

    value = payload[i][j]['value']

i.e. what's going to happen when evaluating `payload[0]['a3']['value']` ?
Perhaps I'm missing something?